prow/cluster/resources/external-secrets/external_secrets_trusted.yaml

```diff
@@ -62,3 +62,16 @@ spec:
     - key: trusted_default_sa-prow-job-github-issues # name of the GCP secret
       name: service-account.json # key name in the k8s secret
       version: latest # version of the GCP secret
+---
```

Comment:
Please add description what is using this secret. You can add  info which preset is dealing whit this secret too.